### DISCUSSION TITLE
[QUESTION/HELP] Use auto dark mode for illustrations

### DISCUSSION THREAD

**Discussion by GhorbaniAli** [tabler/tabler#2017] [Q&A]
Hi, Thanks for this awesome html template and illustrations. I have issue with illustrations with auto dark mode. I faced with this error:
<img width="738" alt="@media" src="https://github.com/user-attachments/assets/e0835cd2-083f-4405-96e9-81a813e1b327">
`@media`

How can I fix this?
(.Net Project - Blazor)

**Comment by ethancrawford** [comment]
@GhorbaniAli - it's not clear from your comments what the actual issue is that you are facing. If you are still having this issue, can you explain it a little more?
